source/_templates/layout.html

```diff
@@ -0,0 +1,21 @@
+{% extends "!layout.html" %}
+{% block content %}
+{% if current_version and latest_version and current_version != latest_version %}
+<div class="rst-content">
+  <div class="admonition warning">
+    <p class="admonition-title">Warning</p>
+    <p>
+      {% if current_version.is_released %}
+      You're reading the manual for Mixxx {{current_version.version}}.
+      If you're using another version, please select to another version in the sidebar.
+      The latest Mixxx release is version {{latest_version.version}}, which you can find <a href="{{ vpathto(latest_version.name) }}">version {{latest_version.version}}</a>.
```

Comment:
This reads a little weird: "The latest Mixxx release is version 2.2, which you can find version 2.2."

maybe "which you can find here."